### PR TITLE
Changed contrast for Tags and Cards options in Note Editor

### PR DESCRIPTION
**Pull request by karthik-anand**
## Purpose / Description
Previously the contrast ratio of the text to the background of the Tags and Cards options in Note Editor was too high. By changing the background color, the options are more readable.

## Fixes
included wideButtonBackgroundColor attr in each of the theme_xyz.xml files. Created another button background file for updated button border colors.

## Approach
Makes it easier to read the options.

## How Has This Been Tested?
UI changes made. No testcases written,

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by criticalAY — COMMENTED** [review]
Please attach the screenshot/s of the changes as they are UI related

**Comment by karthik-anand** [comment]
> Please attach the screenshot/s of the changes as they are UI related

WhatsApp Image 2023-01-30 at 13 50 01
WhatsApp Image 2023-01-30 at 13 50 00

**Review by criticalAY — APPROVED** [review]
Thanks!

File: AnkiDroid/src/main/res/drawable/button_background.xml [hunk 1]
```diff
@@ -1,20 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?attr/buttonRippleColor">
```
Comment (ShridharGoel) [inline AnkiDroid/src/main/res/drawable/button_background.xml:3]: You can consider reverting this indentation change.

File: AnkiDroid/src/main/res/drawable/button_background.xml [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
+android:color="?attr/buttonRippleColor">
     <item>
         <layer-list>
             <!-- Gray shadow of button. -->
-            <item android:left="2dp" android:top="2dp">
-                <shape android:shape="rectangle">
-                    <solid android:color="?attr/buttonShadowColor"/>
-                    <corners android:radius="5dp"/>
+                <shape android:shape="rectangle" >
```
Comment (ShridharGoel) [inline AnkiDroid/src/main/res/drawable/button_background.xml:7]: Extra indentation

**Review by ShridharGoel — APPROVED** [review]
Approving with few comments.

File: AnkiDroid/src/main/res/drawable/button_background.xml (hunk 2 above)
Comment (david-allison) [reply AnkiDroid/src/main/res/drawable/button_background.xml:6]: This still seems incorrect

Comment (karthik-anand) [reply AnkiDroid/src/main/res/drawable/button_background.xml:6]: > This still seems incorrect

Is the indentation incorrect or the code itself?

**Comment by BrayanDSO** [comment]
If someone asks me, I don't like the borders

**Comment by criticalAY** [comment]
> If someone asks me, I don't like the borders

The border that defines the tag and card field?

**Comment by BrayanDSO** [comment]
yep

**Comment by criticalAY** [comment]
I don't have an issue but I agree that these stand out keeping the current AnkiDroid UI that we have maybe we can thin the border a bit. Or remove if we all agree with it.

**Comment by karthik-anand** [comment]
> I don't have an issue but I agree that these stand out keeping the current AnkiDroid UI that we have maybe we can thin the border a bit. Or remove if we all agree with it.

That would be an app wide UI change right? Any changes I can make on my end?

**Comment by david-allison** [comment]
Coming back to this, I also don't like the borders

**Comment by criticalAY** [comment]
Then lets remove them :)

**Comment by david-allison** [comment]
I'll propose closing this PR, given: https://github.com/ankidroid/Anki-Android/wiki/User-Interface-contribution

**Comment by criticalAY** [comment]
> I'll propose closing this PR, given: https://github.com/ankidroid/Anki-Android/wiki/User-Interface-contribution

Cool

**Pull request closed without merging**